insights/config/specs.py

```diff
@@ -94,6 +94,7 @@
     "exim.conf"                 : SimpleFileSpec("etc/exim.conf"),
     "facter"                    : First([CommandSpec("/usr/bin/facter"),
                                     SimpleFileSpec("facts")]),
+    "fc-match"                  : CommandSpec("/usr/bin/fc-match -f 'sans:regular:roman' -sv | grep -e 'family:' -e 'fontformat'"),
```

Comment:
I don't think it's a good idea to add the pipe "|" into the CommandSpec. Better mark it as `large_content` and filter what you want via `add_filter` in the parser file.

Reply:
Please check the command syntax, the `-f` switch requires a format argument and you should be able to specify the elements you want after the search pattern.  For instance here's an example on RHEL7:
```
# fc-match -sv "sans:regular:roman" family fontformat
Pattern has 2 elts (size 16)
	family: "DejaVu Sans"(s)
	fontformat: "TrueType"(w)

Pattern has 2 elts (size 16)
	family: "DejaVu Sans"(s)
	fontformat: "TrueType"(w)

Pattern has 2 elts (size 16)
	family: "DejaVu Sans"(s)
	fontformat: "TrueType"(w)
...
``